### ISSUE TITLE
pileupCaller: cannot parse read base string whith reference skip

### ISSUE THREAD

**Issue by mscbig** [stschiff/sequenceTools#29]
Hello, 
PileupCaller throw an exception when there's a reference skip symbol (> or <) in the pileup file. 

```
pileupCaller: cannot parse read base string >aaaaAAAA
error, called at src/SequenceFormats/Pileup.hs:75:23 in sequence-formats-1.6.3-Xy3K9C32ph8qkWXesYeWT:SequenceFormats.Pileup
```

**Comment by stschiff** [comment]
Oops... that's one I've never encountered. OK, will need to add that.

**Comment by stschiff** [comment]
So, I'm finally looking into this, and I am not sure whether there is really a symbol `>` in the documentation of the Pileup Format. I look here (https://samtools.sourceforge.net/pileup.shtml) and I can't find it. Did you generate this with `samtools mpileup`?

**Comment by stschiff** [comment]
Ah, nevermind, found it: http://www.htslib.org/doc/samtools-mpileup.html#Pileup_Format

**Comment by stschiff** [comment]
Sorry that I never reported back here, but this has been resolved in v1.5.3.
